src/components/faceVerification/standalone/hooks/useFVLoginInfoCheck.js

```diff
@@ -12,7 +12,7 @@ const useFVLoginInfoCheck = navigation => {
       return
     }
 
-    if (fvFlowError || !faceIdentifier) {
+    if (isWebView || fvFlowError || !faceIdentifier) {
```

Comment:
why do we need here to redirect to the error screen? why not show the modal at the same place in fv intro?

Reply:
because fv-standalone jumps over intro screen and goes straight to VerificationScreen